wunderio/frontend-project
# PWS-31: Update npm packages

## Link to ticket

https://wunder.atlassian.net/browse/PSW-31

## Changes proposed in this PR

- [x] Updated ESLINT and its related packages.
- [x] Replaced deprecated `babel-eslint` with `@babel/eslint-parser`. More (https://www.npmjs.com/package/babel-eslint).
- [x] Changed way how `lint-staged` executes linting scripts. There are unknown issue with `npm-run-all` package which halts and never finishes.
- [x] Use current Node LTS - **v16.14** instead of **v12** which includes `package.json.lock` file update.

## How to test

- To test whether this works, you can write an error somewhere in your code and the first time run "npm run prepare", after that try running "npm run lint" and if the file where you wrote the error is staged, then it won't let you commit your changes. It would also tell you the file, the line and the error found
- When running `npm install` it should show `0 vulnerabilities`

## Link to feature environment

https://feature-psw-31.frontend-project.dev.wdr.io/
